vexim/adminaliasaddsubmit.php

```diff
@@ -31,7 +31,7 @@
     $_POST['on_spamassassin'] = 0;
   }
   # If a password wasn't specified, create a randomised 128bit password
-  if (($_POST['clear'] == "") && ($_POST['vclear'] == "")) {
+  if (!validate_password($_POST['clear'], $_POST['vclear'])) {
```

Comment:
No wait, this is wrong. It will create a random password even if the fields were non-empty, but didn't match. I'm sure this wasn't intended.
Sorry, I must have overlooked the context of this line.